### PR TITLE
Update: dropdown & search functionality and css updated

### PR DESCRIPTION
**Pull request by kritika-investwell**

### CODE REVIEW

File: app/media/scss/portfolioCorrelation.scss [hunk 1]
```diff
@@ -186,15 +188,17 @@ $button-color: #5275E9;
...
 
         .dropdownContainer {
-          width: 130px;
+          width: 250px;
           margin: 6px 0 0 0;
+          height: 35px;
           display: flex;
           align-items: center;
           justify-content: first baseline;
+          background-color: white;
```
Comment (rahmatiw) [inline app/media/scss/portfolioCorrelation.scss]: use  hexa code #fff

File: app/media/scss/portfolioCorrelation.scss [hunk 2]
```diff
@@ -293,19 +297,26 @@ $button-color: #5275E9;
...
     border-radius: 5px;
     width: 400px;
+    height: 35px;
     padding: 5px;
     display: flex;
     align-items: center;
     justify-content: space-between;
     font-size: large;
     font-weight: 500;
+    background-color: white;
```
Comment (rahmatiw) [inline app/media/scss/portfolioCorrelation.scss]: same

File: app/media/scss/portfolioCorrelation.scss [hunk 3]
```diff
@@ -186,15 +188,17 @@ $button-color: #5275E9;
         }
 
         .dropdownContainer {
-          width: 130px;
+          width: 250px;
           margin: 6px 0 0 0;
+          height: 35px;
```
Comment (rahmatiw) [inline app/media/scss/portfolioCorrelation.scss]: why add height ?

File: app/media/scss/portfolioCorrelation.scss [hunk 4]
```diff
@@ -293,19 +297,26 @@ $button-color: #5275E9;
...
+    padding: 0 30% 0 38%;
+  }
   .dropdownContainer {
     text-align: left;
+    // margin: 2px 0 0 0;
     border: 1px solid #ccc;
     position: relative;
     border-radius: 5px;
     width: 400px;
+    height: 35px;
```
Comment (rahmatiw) [inline app/media/scss/portfolioCorrelation.scss]: same

File: app/media/scss/portfolioCorrelation.scss [hunk 5]
```diff
@@ -293,19 +297,26 @@ $button-color: #5275E9;
   .errorMsg{
     position: absolute;
     color: red;
-}
+    font-size:15px;
+  }
+  #schemeError{
```
Comment (rahmatiw) [inline app/media/scss/portfolioCorrelation.scss]: @devesh check use of this id

**Review by deveshiwell — APPROVED** [review]

**Pull request merged**